assets/src/blocks/Media/setupMediaElementJS.js

```diff
@@ -3,17 +3,20 @@ import { addLinkTag } from '../../components/useStyleSheet/addLinkTag';
 
 export const setupMediaElementJS = function() {
 	addLinkTag({ href: 'https://cdn.jsdelivr.net/npm/mediaelement@4.2.16/build/mediaelementplayer-legacy.min.css' });
+  const meJSNodes = document.querySelectorAll( '.mejs-video-block' );
```

Comment:
How about moving the early return block to the top, before `addLInkTag`?

Reply:
Ah right missed that one :+1: